backend/src/user/user.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Controller, Delete, Get, Param } from '@nestjs/common';
+import { UserService } from './user.service';
+
+@Controller('user')
+export class UserController {
+  constructor(private readonly userService: UserService) {}
+
+  @Delete(':id')
+  remove(@Param('id') id: Number): string {
```

Comment:
nazwa typu raczej powinna być z małej (prymityw a nie klasa). Chyba że Nest tego nie chce przepuścić, ale to daj znać